include/logger.hpp

```diff
@@ -63,33 +64,36 @@ typedef enum LOG_FACILITY {
   LF_LOCAL5 = 21,
   LF_LOCAL6 = 22,
   LF_LOCAL7 = 23,
-} LOGF_FACILITY;
+};
 
 // Allow for compile-time argument type checking for printf-like functions
 #define printflike(x, y) __attribute__((format(printf, x, y)))
 
 // Manage the logging backend
 bool LOG_syslog_open();
 void LOG_close();
-bool LOG_open(int, const char *);
```

Comment:
Is this a lint or a general convention we should follow ?

Reply:
https://clang.llvm.org/extra/clang-tidy/checks/readability/named-parameter.html
Having named parameters (with names consistent between declaration and definition) helps readability.